docs/deploy.sh

```diff
@@ -0,0 +1,17 @@
+
+#!/usr/bin/env bash
+
+createSearchConfig(){
+    # Get the data
+    MEILI_CONFIG_DEST=public/scripts/xss/dist/config/config.json
+    MEILI_INDEX="docs"
+    MEILI_TOKEN=$(curl -s -H "X-Meili-API-Key: $PLATFORM_PROJECT_ENTROPY" -X GET "search.internal/keys" | jq -r ".public")
+    MEILI_URL=$(echo $PLATFORM_ROUTES | base64 --decode | jq -r 'to_entries[] | select(.value.id == "search") | .key')
+    # Delete config file in the mount if it exists
+    [ ! -e $MEILI_CONFIG_DEST ] || rm $MEILI_CONFIG_DEST
```

Comment:
```suggestion
    rm -f $MEILI_CONFIG_DEST
```

I think this does the same thing (rm and don't error if it's not there), but is shorter.